File: java/client/src/org/openqa/selenium/devtools/network/types/MixedContentType.java
package org.openqa.selenium.devtools.network.types;

/**
 * A description of mixed content (HTTP resources on HTTPS pages), as defined by
 * https://www.w3.org/TR/mixed-content/#categories
 */
public enum MixedContentType {

  BLOCKABLE("blockable"),
  OPTIONALLY_BLOCKABLE("optionally-blockable"),
  NONE("none");

  private String type;

  MixedContentType(String type) {
    this.type = type;
  }

  public String getType() {
    return type;
  }
}

File: dotnet/src/webdriver/DevTools/AutoGenerated/Security/MixedContentType.cs
namespace OpenQA.Selenium.DevTools.Security
{
    using Newtonsoft.Json;
    using Newtonsoft.Json.Converters;
    using System.Runtime.Serialization;

    /// <summary>
    /// A description of mixed content (HTTP resources on HTTPS pages), as defined by
    /// https://www.w3.org/TR/mixed-content/#categories
    /// </summary>
    [JsonConverter(typeof(StringEnumConverter))]
    public enum MixedContentType
    {
        [EnumMember(Value = "blockable")]
        Blockable,
        [EnumMember(Value = "optionally-blockable")]
        OptionallyBlockable,
        [EnumMember(Value = "none")]
        None,
    }
}